src/parser/test/ParserTest.cpp

```diff
@@ -940,4 +940,22 @@ TEST(Parser, ReentrantRecoveryFromFailure) {
     }
 }
 
+TEST(Parser, Distinct) {
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 1 over friend "
+                            "YIELD DISTINCT friend.name as name, friend.age as age";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 1 OVER like "
+                            "| GO FROM $-.id OVER like | GO FROM $-.id OVER serve "
+                            "YIELD DISTINCT serve._dst, $$.team.name";
```

Comment:
syntax error: yield col1, distinct col2

Reply:
That is what we want, and I add this test case in parser.